src/cirrus/builtins/functions/process/lambda_function.py

```diff
@@ -1,65 +1,88 @@
 import json
-import os
 
+from cirrus.lib.errors import NoUrlError
 from cirrus.lib.process_payload import ProcessPayload, ProcessPayloads
-from cirrus.lib.utils import dict_merge
-from cirrus.lib.logging import get_task_logger
+from cirrus.lib import utils
+from cirrus.lib.logging import get_task_logger, defer
 
-logger = get_task_logger('lambda_function.process', payload=tuple())
 
-# Default PROCESSES
-# TODO: put this configuration into the cirrus.yml
-with open(os.path.join(os.path.dirname(__file__), 'processes.json')) as f:
-    PROCESSES = json.loads(f.read())
+logger = get_task_logger('function.process', payload=tuple())
+
+
+def is_sqs_message(message):
+    return (
```

Comment:
```suggestion
    return message.get('eventSource') == 'aws:sqs'
```

Reply:
Ha, that was terrible code. Thanks for the much better rewrite.